experimental/origin-dapp2/src/pages/_ListingBadge.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+
+const ListingBadge = ({ status, featured }) => {
+  if (status === 'sold') {
+    return <div className="badge badge--sold">sold</div>
+  }
+  // TODO: Unavailable (multi unit, pending)
+  // TODO: Pending (single unit, pending)
```

Comment:
I started out building the logic for this by looking into the events, but it seemed too much business logic in there. I think we should consider returning a `pending: Boolean` in the graphql as well. That should probably be its own PR and then this can be implemented.

Reply:
How about we expand the `status` field on a listing to include 'unavailable', 'pending', 'sold' and 'soldout'. Agree we can do this in a separate PR.

Reply:
I'd prefer that as well! Created #1163 for that.